### DISCUSSION TITLE
Integrated Webhooks functionality

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#3806] [Feature Requests]
### Suggest an idea for ExpressionEngine

Although the Weebhooks add-on by Blair Liikala is good, it's still a third-party add-on with all the associated risks.

I believe that the integration of Webhooks functionality into the core of ExpressionEngine can be a big step forward for ExpressionEngine, keeping the CMS competitive and aligned with modern web development practices. 

Webhooks have become an essential component of the modern web, enabling seamless and efficient communication between websites and various third-party services and applications.

With Webhooks support, ExpressionEngine users can automate a wide range of tasks and streamline their workflow. They can trigger events in real-time or near real-time, such as updating a CRM system when a new user registers on the website, sharing content on social media platforms automatically, or sending notifications to team members when specific content is published. Or anything else, using automation platforms like IFTTT or Zapier.

**Comment by blairliikala** [comment]
Yep, I would agree here.  Maybe make it part of the channel config.

Though I do think there are some other priorities that should come first that are not able to be filled-in by an add-on.
